Makefile

```diff
@@ -15,3 +28,13 @@ release: test
 
 clean:
 	rm -rf ./dist
+
+CI_REGISTRY_IMAGE ?= gomeeseeks/meeseeks-box
+
+build_docker:
+	docker build -t $(CI_REGISTRY_IMAGE):$(VERSION) -f Dockerfile .
+	if [ -n "$$CI_REGISTRY" ]; then \
+	  docker login --username $$CI_REGISTRY_USER --password $$CI_REGISTRY_PASSWORD $$CI_REGISTRY && \
+	  docker push $(CI_REGISTRY_IMAGE):$(VERSION) && \
+	  docker logout $$CI_REGISTRY; \
+	fi
```

Comment:
Check my previous comment, this should not be necessary as part of the makefile (the docker build at least), then we could extract the gitlab-ci specifics into the gitlab-ci file leaving this one provider agnostic.